windows/RNSensors.sln

```diff
@@ -0,0 +1,59 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This is the solution file of the project, to add dependencies, etc... This is not needed by the devs will integrate the module but easier for the development of the module itself :)